athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/handlers/GlueMetadataHandler.java

```diff
@@ -199,6 +202,14 @@ protected String getCatalog(MetadataRequest request)
     {
         String override = System.getenv(CATALOG_NAME_ENV_OVERRIDE);
         if (override == null) {
+            if (request.getContext() != null) {
```

Comment:
Looks good, but can you add some tests for `getCatalog`?

Reply:
good looks, done